se/se_epub_build.py

```diff
@@ -195,6 +195,7 @@ def build(self, run_epubcheck: bool, check_only: bool, build_kobo: bool, build_k
 					file.truncate()
 
 		# Update the release date in the metadata and colophon.
+		last_updated = None
```

Comment:
I don't see this getting set to a commit timestamp anywhere, won't it just always be `None` in practice?